src/settings-ui/Settings.UI/ViewModels/DashboardViewModel.cs

```diff
@@ -0,0 +1,574 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using global::PowerToys.GPOWrapper;
+using Microsoft.PowerToys.Settings.UI.Library;
+using Microsoft.PowerToys.Settings.UI.Library.Helpers;
+using Microsoft.PowerToys.Settings.UI.Library.Interfaces;
+using Microsoft.PowerToys.Settings.UI.Services;
+using Microsoft.PowerToys.Settings.UI.Views;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+
+namespace Microsoft.PowerToys.Settings.UI.ViewModels
+{
+    public class DashboardViewModel : Observable
+    {
+        public Func<string, int> SendConfigMSG { get; }
+
+        public ObservableCollection<DashboardListItem> ActiveModules { get; set; } = new ObservableCollection<DashboardListItem>();
+
+        public ObservableCollection<DashboardListItem> AllModules { get; set; } = new ObservableCollection<DashboardListItem>();
+
+        public bool UpdateAvailable { get; set; }
+
+        private List<DashboardListItem> _allModules;
+
+        private ISettingsRepository<GeneralSettings> _settingsRepository;
+        private GeneralSettings generalSettingsConfig;
+        private Windows.ApplicationModel.Resources.ResourceLoader resourceLoader = Helpers.ResourceLoaderInstance.ResourceLoader;
+
+        public DashboardViewModel(ISettingsRepository<GeneralSettings> settingsRepository, Func<string, int> ipcMSGCallBackFunc)
+        {
+            _settingsRepository = settingsRepository;
+            generalSettingsConfig = settingsRepository.SettingsConfig;
+            generalSettingsConfig.AddEnabledModuleChangeNotification(ModuleEnabledChangedOnSettingsPage);
+
+            // set the callback functions value to hangle outgoing IPC message.
+            SendConfigMSG = ipcMSGCallBackFunc;
+
+            _allModules = new List<DashboardListItem>();
+
+            GpoRuleConfigured gpo;
+            if ((gpo = GPOWrapper.GetConfiguredAlwaysOnTopEnabledValue()) != GpoRuleConfigured.Disabled && gpo != GpoRuleConfigured.Enabled)
+            {
+                _allModules.Add(new DashboardListItem() { Label = resourceLoader.GetString("AlwaysOnTop/ModuleTitle"), IsEnabled = generalSettingsConfig.Enabled.AlwaysOnTop, Tag = "AlwaysOnTop", Icon = "ms-appx:///Assets/Settings/FluentIcons/FluentIconsAlwaysOnTop.png", EnabledChangedCallback = EnabledChangedOnUI, DashboardModuleItems = GetModuleItemsAlwaysOnTop() });
+            }
```

Comment:
If the policy is enabled the module is available and the user can't disable it. If the polivy is disabled the module is disabled and the user can't enable it.

Maybe we could do it like MS Edge and show a lock nearby the on/off button.

Reply:
Hi @htcfreek,
I like your proposal. Now as you mentioned it I've realised that a module, which is GPO-switched-on wouldn't even appear in the lists, which is really a mistake. Thanks for the comment, I will update the code.

Reply:
I saw the screenshot and like it so much. I think we should port the layout and behavior over to Tray Flyout > All Apps list.